### ISSUE TITLE
feedback

### ISSUE THREAD

**Issue by yotuke** [gusye1234/pytorch-light-gcn#46]
Your codes  styles are so ugly, names of variants are too difficult to understand.

**Comment by gusye1234** [comment]
You are right actually😂. This is a project really long time ago and I have not intent to continue maintain it.
